diff -ruPN u-boot-68c4faa3f8204593f60dadfb560abe58b1899556/include/configs/rockchip-common.h u-boot-new/include/configs/rockchip-common.h
--- u-boot-68c4faa3f8204593f60dadfb560abe58b1899556/include/configs/rockchip-common.h	2020-03-30 08:52:08.000000000 +0200
+++ u-boot-new/include/configs/rockchip-common.h	2022-11-30 11:02:07.140036860 +0100
@@ -138,8 +138,6 @@
 	"boot_android ${devtype} ${devnum};"
 #else
 #define RKIMG_BOOTCOMMAND			\
-	"boot_android ${devtype} ${devnum};"	\
-	"bootrkp;"				\
 	"run distro_bootcmd;"
 #endif
 
